src/FerriteGmsh.jl

```diff
@@ -184,8 +215,9 @@ function saved_file_to_grid(filename::String; domain="")
 
     boundarydict = toboundary(facedim)
     facesets = tofacesets(boundarydict, elements)
-    gmsh.finalize()
-        
+
+    should_finalize && finalize()
```

Comment:
just asking if I get it right: doesn't need to finalize if it's initialized in the scope of the function because you register the atextit in the finalize call?

Reply:
True. The idea was that if this function was responsible for the initialization, then it should also finalize. I think it is nice to do this directly in that case, since otherwise there might be some resources allocated for no use until Julia quits. Doing it conditionally is to make sure we only finalize if we were responsible for the initialize.